### ISSUE TITLE
Missing translations for tags under minecraft namespaces.

### ISSUE THREAD

**Issue by vfyjxf** [neoforged/NeoForge#1271]
**Minecraft Version:** 1.21

**NeoForge Version:** 21.0.57-beta

**Description of issue:**
Missing translations for tags under minecraft namespaces.
[en_us,json](https://github.com/neoforged/NeoForge/blob/1.21.x/src/generated/resources/assets/c/lang/en_us.json ) only provides tag translations under common namespace

**Comment by TelepathicGrunt** [comment]
Yeah this was originally intentional. Translating Minecraft’s stuff is an idea that neoforge should look into. Like translating dimension names and structure names too. Just needs someone to think up the standard of what to translate in Mc and how to setup Neo to do so

**Comment by RaymondBlaze** [comment]
> Just needs someone to think up the standard of what to translate in Mc and how to setup Neo to do so

The translation project should be separate from Neo's crowdin project, and we should probably work it out with the Fabric community and hopefully the vanilla Minecraft translation community.
Also, is a translation project set up for the conventional tags yet? By far I haven't seen Neo pulling in translation for tags for other languages, so I guess it's not a thing yet?

**Comment by TelepathicGrunt** [comment]
@RaymondBlaze There is no common tag repo. Some people tried for it but due to disagreements on ownership and governance and how decisions would be made, the idea never fell through.

**Comment by Matyrobbrt** [comment]
This would be a big task for Neo to undertake by itself as there are hundreds of tags, and the effort might be duplicated between loaders. While some sort of collaboration would be beneficial, there are currently no plans regarding this and these initiatives are unfortunately not as easy to set up as it may seem. I'll be closing this for now
